### DISCUSSION TITLE
How to use dotenv properly

### DISCUSSION THREAD

**Discussion by kannaiah** [quasarframework/quasar#13755] [General - Components / Directives / etc]
I have VUE frontend using following environment variables being used

    VUE_APP_DOMAIN_DEV=localhost
    VUE_APP_DOMAIN_STAG=<>
    VUE_APP_DOMAIN_PROD=<>
    VUE_APP_NAME=<>
    VUE_APP_ENV=development

I would like to use the same environment variables and same file as I would be having two frontends for sometime going forward.

If I use dotenv to load env variables as mentioned in https://quasar.dev/quasar-cli-vite/handling-process-env#vites-built-in-env
`/quasar.config.js`
    
    build: {
        env: require('dotenv').config().parsed
    }

Are `process.env` values available in `boot.js`
I would like to change `api` url based on environment variables.

Thank you

**Comment by pratik227** [comment]
@kannaiah 

Yes You can Directly use the process.env in boot files.

Example - `process.env.API`

Doc - https://quasar.dev/quasar-cli-webpack/handling-process-env#adding-to-process-env
